src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-update-firmware-node.ts

```diff
@@ -119,7 +119,7 @@ class DialogZWaveJSUpdateFirmwareNode extends LitElement {
       <ha-form
         .hass=${this.hass}
         .data=${{ firmware_target: this._firmwareTarget }}
-        .schema=${[schema]}
+        .schema=${[firmwareTargetSchema]}
```

Comment:
We are now still creating an array every render, this makes an equality check fail: `[] !== []` causing a re-render in `ha-form`